api/users/models.py

```diff
@@ -121,6 +121,18 @@ def __str__(self):
     def subscribe_to_mailing_list(self):
         mailer_lite.subscribe(self)
 
+    @hook(AFTER_UPDATE, when="email", has_changed=True)
+    def send_warning_email(self):
+        from users.tasks import send_email_changed_notification_email
```

Comment:
We should avoid using local imports like this as much as possible but I can't see an easy way to avoid it here!

Reply:
Yes, I will try to avoid it as much as possible.